### PR TITLE
Add Pantheon to Featured Page

### PR DESCRIPTION
**Pull request by alexfornuto**
Our [repository](https://github.com/pantheon-systems/documentation) has the `docsthursday` tag and we announced our involvement on our [forum](https://discuss.pantheon.io/t/pantheon-docsthursday/227).

### CODE REVIEW

**Comment by FelicianoTech** [comment]
@alexfornuto Hey, I enabled CI builds for forked PRs. Since this is two commits, do you mind rebasing and force pushing this which will clean this up while more importantly, kicking off a build?

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 src/public
 src/resources
+*.un~
```
Comment (FelicianoTech) [inline .gitignore:3]: What is this for?

Comment (alexfornuto) [reply .gitignore:3]: vim undo files.

Comment (FelicianoTech) [reply .gitignore:3]: Ah okay.

**Pull request merged**